arch/arm/soc/nordic_nrf5/nrf51/soc.c

```diff
@@ -34,6 +36,17 @@ static bool ftpan_59(void);
 
 uint32_t SystemCoreClock __used = __SYSTEM_CLOCK;
 
+void relocate_vector_table(void)
```

Comment:
Shouldn't this function be in a more generic file shared by all Cortex-M0 boards that do not have hw vector table relocation?

Reply:
I assume that this is the point where forwarding actually begins, from the bootloader to the application image, so it really makes sense that we have this somewhere in `arch/arm/cortex_m/`

Reply:
Actually, why add this if there's an identical version already present here:

https://github.com/zephyrproject-rtos/zephyr/blob/master/arch/arm/core/cortex_m/prep_c.c#L30

Reply:
Good idea, will do.